src/lib/datepicker/datepicker.ts

```diff
@@ -289,6 +289,11 @@ export class MatDatepicker<D> implements OnDestroy, CanColor {
     }
   }
 
+  /** Public method to select a date in the Datepicker */
+  select(date: D): void {
```

Comment:
@mmalerba I added a public setter 
(https://github.com/angular/material2/pull/11821#discussion_r199946419)

Reply:
done in 
https://github.com/angular/material2/pull/11821/commits/cb84cbe9c2fc5a02da3ba045ec1d91d4afef55e8